lib/srv/alpnproxy/helpers_test.go

```diff
@@ -263,7 +263,7 @@ func mustCallHTTPSServerAndReceiveCode(t *testing.T, addr string, client http.Cl
 	require.Equal(t, expectStatusCode, resp.StatusCode)
 }
 
-func mustStartHTTPServer(t *testing.T, l net.Listener) {
+func mustStartHTTPServer(_ *testing.T, l net.Listener) {
```

Comment:
I think it may be an idea to call t.Helper() instead. SAmein mustStartLocalProxy below

Reply:
I just removed it since it's not used for anything